woocommerce/woocommerce
# [Product Block Editor]: relocate block actions in the sections area

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR is a follow-up of https://github.com/woocommerce/woocommerce/pull/43540, which updates section actions for the following blocks:

* General / Images
* General / Downloads
* Organization / Attributes
* Variations / Options
* Linked products





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new Product Editor
2. Check the action buttons of the following blocks
  * General / Images
  * General / Downloads
  * Organization / Attributes
  * Variations / Options
  * Linked products
3. Confirm the buttons work as expected
4. Confirm the buttons are properly aligned with the section header

<img width="722" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/ae7944bf-99d0-420e-9f3c-875fadfc5dd8">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
